src/core/DrumSynth.cpp

```diff
@@ -162,17 +164,20 @@ float DrumSynth::waveform(float ph, int form)
 }
 
 
-int DrumSynth::GetPrivateProfileString(const char *sec, const char *key, const char *def, char *buffer, int size, const char *file)
+int DrumSynth::GetPrivateProfileString(const char *sec, const char *key, const char *def, char *buffer, int size, QString file)
 {
-    ifstream is;
+    stringstream is;
     bool inSection = false;
     char *line;
     char *k, *b;
     int len = 0;
 
     line = (char*)malloc(200);
 
-    is.open (file, ifstream::in);
+    QFile f(file);
+    f.open(QIODevice::ReadOnly);
+    QByteArray dat = f.readAll().constData();
+    is.str(string(dat.constData(), dat.size()));
```

Comment:
The casual reader will see this and think "we're reading the entire file into a string, and then operating on it only as a stream? Why not directly open the file with `ifstream`?" and will be tempted to change this back to how it was previously. I think there should be some [short] comment explaining why we _must_ open the file with `QFile`, possibly linking back to this github PR.